common/src/main/java/com/wynntils/models/activities/markers/ActivityMarkerProvider.java

```diff
@@ -35,7 +36,8 @@ public void setSpawnLocation(Location spawnLocation) {
                                 Texture.QUESTS_SCROLL_ICON,
                                 CommonColors.WHITE,
                                 CommonColors.WHITE,
-                                CommonColors.WHITE),
+                                CommonColors.WHITE,
+                                null),
```

Comment:
Maybe put `ACTIVITY_LOCATION_NAME` here, if the config is enabled?

Reply:
Well, then it will show "Activity Location" text overlapped with name of the activity. Tho I think I get it now, it is like placing 2 different markers in the same place, one is coming from `setSpawnLocation` and second `setTrackedActivityLocation`. Is there a necessity for it to do that?
![image](https://github.com/user-attachments/assets/fe759e8a-9727-4475-acab-40264c777756)

Reply:
We set duplicated markers? That's a separate issue.. I guess ignore this change, and we'll fix it eventually.